### PR TITLE
Fix visibility of text descenders in Tile component [experiments]

### PR DESCRIPTION
**Pull request by ThomasMichon**
### Overview

This change fixes the visibility of text descenders in the nameplate of the `Tile` component.
Descenders such as 'g' were being cut off, which is not ideal.
This change switches the layout within the regions to use flexbox for vertical alignment, and uses negative margins and larger heights to provide clearance for the `New` signal's "glimmer" effect while preserving ellipsis overflow.

### CODE REVIEW

File: packages/experiments/src/components/Tile/Tile.scss [hunk 1]
```diff
@@ -253,54 +253,61 @@ button.check {
...
+  @include margin-left(-8px);
+  @include padding-left(8px);
+
   &,
   .isLarge & {
     font-size: $ms-font-size-m;
-    height: 17px;
-    line-height: 17px;
+    height: 30px;
+    margin-top: -7px;
```
Comment (battletoilet) [inline packages/experiments/src/components/Tile/Tile.scss]: These adjustments seem a bit random, can you specify what negative margins are doing?

File: packages/experiments/src/components/Tile/Tile.scss [hunk 2]
```diff
@@ -253,54 +253,61 @@ button.check {
...
+    margin-bottom: -4px;
   }
 
   .isSmall & {
     font-size: $ms-font-size-s;
-    height: 14px;
-    line-height: 14px;
+    height: 24px;
+    margin-top: -5px;
+    margin-bottom: -3px;
```
Comment (battletoilet) [inline packages/experiments/src/components/Tile/Tile.scss]: Same here.

File: packages/experiments/src/components/signals/Signals.scss [hunk 3]
```diff
@@ -77,6 +77,10 @@
 }
 
 .mention {
+  position: relative;
+
+  top: 0.2em;
```
Comment (dzearing) [inline packages/experiments/src/components/signals/Signals.scss]: this is very odd css...

Comment (ThomasMichon) [reply packages/experiments/src/components/signals/Signals.scss]: Well, if someone could just fix the icon font so the glyphs lined up better that would be nice!

Comment (dzearing) [reply packages/experiments/src/components/signals/Signals.scss]: can you add a comment then about your reasoning here?

File: packages/experiments/src/components/Tile/Tile.scss (hunk 1 above)
Comment (dzearing) [reply packages/experiments/src/components/Tile/Tile.scss]: Add comments...as someone coming in fresh to this css I would be very leery of this and quick to redo it. But there are probably good reasons here...

**Pull request merged**